### DISCUSSION TITLE
Is there a way to force page breaks in the markdown source that is respected by mdbook-pdf?

### DISCUSSION THREAD

**Discussion by GeoffMurray** [HollowMan6/mdbook-pdf#9] [Q&A] [answered]
I see the page break on chapters (which I might be able to use for my purposes), but is there an alternate way to request a page break in the MD source file that flows all the way to mdbook-pdf, and allows me to put a page break at a specified point in the text? 

Thanks.

**Comment by HollowMan6** [comment, marked as answer]
Hi! Actually the page break on chapters is a default behaviour of mdbook, you can turn it off by setting `page-break = false` for the `[output.html.print]`: https://rust-lang.github.io/mdBook/format/configuration/renderers.html#outputhtmlprint Anything that are present in the generated `print.html` should be followed by the `mdbook-pdf`.

I'm not sure if there is an alternate way to request a page break in the MD source file manually for mdbook, maybe you can simply add the [html element](https://www.buildableweb.com/how-to-add-page-breaks-to-html-in-articles) in the MD file to do so, or just ask for some kind of way at [mdbook's issue](https://github.com/rust-lang/mdBook/issues).

**Comment by GeoffMurray** [comment]
In case anyone else is looking for it, we found this worked: 

```html
<div style="page-break-before:always">&nbsp;</div>
<p></p>
```

The ```&nbsp;``` element was critical to generate the actual break. 
The ```<p></p>``` resets processing. Without it, if the next line contains header markdown processing, the header level was not recognized.

**Reply by kameshwari1991** [reply]
After a day of searching 
'how to insert page-break in pdf file through markdown in jupyter-notebook'.....
converting it through nbconvert (to html) and wkhtmltopdf(html to pdf).....
your solution actually worked...it has inserted many page breaks....but it did what I wanted (image being displayed on two pages, now gets displayed on a fresh page)...thanks a lot!!!!

**Reply by npj210mlk** [reply]
Awesome - thanks!
